README.md

```diff
@@ -35,6 +35,15 @@ yarn add near-ca
 
 ## Usage
 
+Create a .env file. Make sure the environment variables are defined in the .env file. You can view the required variables in the .env.example
+
+Fund your address with some sepolia ETH
+
+Run 
+```bash
+npx ts-node examples/send-eth.ts
```

Comment:
An example script (the opensea example) is already indicated above. Perhaps it would be easier to change that line above to send-eth (since the open sea example is a bit more involved).